README.md

```diff
@@ -447,6 +447,67 @@ There are two paths to working with this DevContainer setup.
 
 2. Run `docker` directly. Run `docker build -t freshli-cli-dev .devcontainer/` to build the container. Then you'll be able to run `docker run --rm -it -v $PWD:/code -w /code freshli-cli-dev bash` to create a shell session inside of a running container with everything set up for you. (Note, you may need to run `bundle install` when you first start the container to install the ruby-based dependencies. This step is performed for you if you use the `devcontainer` CLI to open a Visual Studio Code instance.)
 
+## Production container
+
+### Building for local use
+
+A production-ready container can be created from the `Dockerfile` in this project by running:
+
+```bash
+docker build -t freshli-cli .
+```
+
+You can then run the container with:
+
+```bash
+docker run --rm freshli-cli agents detect
+```
+
+You should see output that looks similar to:
```

Comment:
I saw your message in Slack yesterday, it works on my machine as well! :D 
![image](https://user-images.githubusercontent.com/7047894/189088576-6494d4be-fb93-4132-9baf-9ea8b6980432.png)

Reply:
Yay!